internal/api/api.go

```diff
@@ -0,0 +1,8 @@
+package api
+
+import (
+	_ "embed"
+)
+
+//go:embed openapi.json
+var Api []byte
```

Comment:
This is nitpicky but I might change the name of this variable to ```ApiSpec``` or something a bit more descriptive.  I'm not sure Api communicates that its really the contents of the openapi spec file.  Just my $0.02...

Reply:
BTW, the ```embed``` approach here is pretty slick!  :+1:   :)

Reply:
sweet. I'll do that. That's smart